src/liquid/utils/ariaDisabled.ts

```diff
@@ -0,0 +1,3 @@
+// 'true', '1', 'yolo' etc. correspond to an aria disabled element, but not 'false'.
+export const isAriaDisabled = (ariaDisabledValue: string | undefined) =>
```

Comment:
Because the argument is not optional. It must be provided. With this notation you get an error message from TS if you use `isAriaDisabled()`. With the _optional_ notation you are allowed to "forget" to provide an argument. So this is stricter.